site/_layouts/action-network-petitions.html

```diff
@@ -107,7 +107,7 @@ <h2 id="petition-title">{{ page.action_network_title }}</h2>
 
     <label for="opt-in">
       {% if page.opt_in == true %}
-      <input type="hidden" name="subscription[group]" value="13162" form="petition-form"/>
+      <input type="hidden" name="subscription[group]" value="13162" form="petition-form" id="opt-in"/>
```

Comment:
Gives the submission text the ID it had been looking for. (I think this a merge-conflict-related regression)